docs/src/managing-packages.md

```diff
@@ -82,6 +82,9 @@ we can explicitly track that branch (or commit) by appending `#branchname` (or `
 
 The status output now shows that we are tracking the `master` branch of `Example`.
 When updating packages, we will pull updates from that branch.
+Notice that if instead of a branch name we specified an exact commit id, e.g.
```

Comment:
!!! note
    If we would specify a commit id instead of a branch name, e.g.
    `add Example#025cf7e`, then we would effectively "pin" the package
    to that commit. This is because the commit id always point to the same
    thing unlike a branch, which may be updated.